api/python/quilt3/workflows/config-1.schema.json

```diff
@@ -0,0 +1,77 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://quiltdata.com/workflows/config/1",
+  "type": "object",
+  "required": [
+    "version",
+    "workflows"
+  ],
+  "additionalProperties": false,
+  "properties": {
+    "version": {
+      "const": 1
```

Comment:
Should we use `string` instead of `number` for config version?

Reply:
My opinion: `strings` for version is better. It's more reliable, and we can use even semver

Reply:
I think it's more flexible, so I made it a `string`. Though I'm not sure how semver could be applied for schemas.